README.md

```diff
@@ -37,18 +44,25 @@ The purpose of this application is to sync ArchivesSpace records to FOLIO.  It a
    ```bash
    cp config/templates/folio_sync.yml config/folio_sync.yml
    ```
-   Edit `config/folio_sync.yml` to include settings specific to the script:
+   Edit `config/folio_sync.yml` to include settings specific to the script. The script supports multiple instances.
    ```yaml
    development:
-     marc_download_directory: <%= Rails.root.join('tmp/development/marc_files') %>
-     default_sender_email_address: "no-reply@example.com"
-     marc_sync_email_addresses:
-       - "admin@example.com"
-   ```
+     default_sender_email_address: "folio-sync@abc.com"
+     marc_download_base_directory: <%= Rails.root.join('tmp/development/downloaded_files') %>
+     instances:
```

Comment:
Would you mind if we change `instances` to `archivesspace_instances` here?  I just thought of this now while reviewing, but when this FolioSync app eventually handles non-ArchivesSpace syncing (like FOLIO to Hyacinth), it'll probably be good for the folio_sync.yml file to be specific about this being an ArchivesSpace-specific configuration option.